### PR TITLE
Adds icons and logos for HDFury VRROOM custom integration.

### PR DESCRIPTION
**Pull request by vingenuity**
## Proposed change

Adds icons and logos at both DPIs for the HDFury brand and VRROOM custom component I am currently working on.

## Type of change

- [ ] Add a new logo or icon for a new core integration
- [ ] Add a missing icon or logo for an existing core integration
- [x] Add a new logo or icon for a custom integration (custom component)
  - [ ] I've opened up a PR for my custom integration on the [Home Assistant
    Python wheels repository](https://github.com/home-assistant/wheels-custom-integrations)
- [ ] Replace an existing icon or logo with a higher quality version
- [ ] Removing an icon or logo

## Additional information

- This PR fixes or closes issue: fixes #
- Link to code base pull request: 
- Link to documentation pull request: 
- Link to integration documentation on our website: 

## Checklist

- [x] The added/replaced image(s) are **PNG**
- [x] Icon image size is 256x256px (`icon.png`)
- [x] hDPI icon image size is 512x512px for  (`icon@2x.png`)
- [x] Logo image size has min 128px, but max 256px, on the shortest side (`logo.png`)
- [x] hDPI logo image size has min 256px, but max 512px, on the shortest side (`logo@2x.png`)

### CODE REVIEW

**Review by frenck — APPROVED** [review]
This PR adds two different integrations. Please split it up into separate PRs (one integration per PR).

Thanks 👍 

../Frenck

**Review by frenck — CHANGES_REQUESTED** [review]
Oops, wrong button, let me try again.

**Comment by frenck** [comment]
Since there was no response at all, I'm declaring this PR stale and closing it.
Feel free to re-open this PR when you feel like that is the appropriate thing to do.

**Comment by vingenuity** [comment]
Sorry frenck, have had a really busy couple of weeks that haven't allowed me to do any extra dev work.

After thinking about it, I have decided to try and make the VRROOM plugin official, but I haven't gotten the process started yet.

As soon as the official plugin seems like it is going to pass muster, I will reopen this CL under the official folders and split the HDFury logos into a separate CL as requested.

Thanks for the review and letting me know how things work around here.
